README.md

```diff
@@ -9,13 +9,14 @@ Please report bugs via the
 
 The main repo:
 
-```bash
```

Comment:
Could you please keep `bash` in place here and below? Alternatively, replace `bash` with `sh`, there is nothing bash-specific.

Reply:
`cd` now appears in colour